django_freeradius/tests/base/test_api.py

```diff
@@ -181,7 +185,7 @@ def assertAcctData(self, ra, data):
         to ensure they match
         """
         for key, value in data.items():
-            if key == 'status_type':
+            if key in ('status_type', 'framed_ipv6_address'):
```

Comment:
Because of the way django treats ipv6 addresses, it keeps converting `::ffff:c0a8:b603` into `::ffff:192.168.182.3`, so I could only do this.